node/geth_node.go

```diff
@@ -497,49 +487,6 @@ func createWakuService(ctx *node.ServiceContext, wakuCfg *params.WakuConfig, clu
 	return w, nil
 }
 
-// activateIncentivisationService configures Whisper and adds it to the given node.
-func activateIncentivisationService(stack *node.Node, config *params.NodeConfig) (err error) {
```

Comment:
Why are we removing this tho? I don't see this mentioned in PR description.

Reply:
Last point in the PR description